mitodl/mit-open
# Remove an unused dependency

### What are the relevant tickets?




N/A

### Description (What does it do?)

This preempts this renovate PR: https://github.com/mitodl/mit-open/pull/1553

Best I can guess this was a transient dependency we pinned to a specific version years ago in open-discussions. Either we removed the package that depended on it or that package changed their requirements, but either way, I ran `poetry remove importlib_metadata` and it's no longer getting installed and everything works.

### How can this be tested?

- You should be able to `docker compose build` and then `docker compose up`, the app should generally work. 
- You can run `poetry show --tree | grep importlib_metadata` to verify the dependency is completely gone.
